src/utilities/camelcase.js

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+// This code is a copy of the code made available by https://www.npmjs.com/package/camelcase
+// Since the original module isn't supported by create-react-app we are using the code manually here
```

Comment:
Can you explain this sentence? Are we removing the camelcase-Object and using camelcase? I don't understand what create-react-app has to do with this

Reply:
We are removing the camelcase-object package but using the camelcase package code which which was being used by camelcase-object. Because of the way camelcase bundling is configured, it doesn't support the camelcase package since it was packaged to be used by node, not the browser. This is a workaround to use the code and circumvent the packaging issue.